authentication/oidc.go

```diff
@@ -434,3 +434,7 @@ func (a oidcAuthenticator) checkAuth(ctx context.Context, token string) (context
 
 	return ctx, "", http.StatusOK, codes.OK
 }
+
+func (a oidcAuthenticator) LoginPath(tenant string) string {
+	return strings.ReplaceAll("/oidc/{tenant}/login", "{tenant}", tenant)
```

Comment:
Maybe `path.Join("/oidc", tenant, "login")` would be better here?

Reply:
I am happy to make this change.

We currently have `handlerPrefix = "/oidc/{tenant}"` and `return "/oidc/{tenant}", a.handler`.  My thought was that it was nice to keep a similar appearance in related code.  Eventually those two fragments and my new piece should be merged.

Similar code exists in _authentication/openshift.go_.  Let me know what you think and I will make it happen.

Reply:
I'd still say `path.Join` is more common in our code base as well as easier on the eyes, it's a minor point though.